### PR TITLE
feat: add logPlaygroundUrl in screen

### PR DESCRIPTION
**Pull request by balavishnuvj**
Based on discussion at #780

### CODE REVIEW

**Comment by balavishnuvj** [comment]
Do let me know what you think of this. This mostly PRd version of what @smeijer had suggested.

File: src/screen.js [hunk 1]
```diff
@@ -1,23 +1,41 @@
...
+}
 
 const debug = (element, maxLength, options) =>
   Array.isArray(element)
     ? element.forEach(el => logDOM(el, maxLength, options))
     : logDOM(element, maxLength, options)
 
+const logTestingPlaygroundURL = (element = getDocument().body) => {
+  console.log(`Open this URL in your browser\n\n${getPlaygroundUrl(element)}`)
+}
```
Comment (marcosvega91) [inline src/screen.js:28]: As for the previous comment. What do you think of this?

```suggestion
const logTestingPlaygroundURL = (element = getDocument().body) => {
  if ('innerHTML' in element) {
    if (element.innerHTML === '')
      console.log(
        `The provided element(${element.nodeName}) doesn't have any children.`,
      )
    console.log(`Open this URL in your browser\n\n${getPlaygroundUrl(element)}`)
  } else {
    console.log(`The element you're providing isn't a valid DOM element.`)
  }
}

**Review by marcosvega91 — CHANGES_REQUESTED** [review]
Hi @balavishnuvj thanks for this implementation 💯 . I have added some comments

**Comment by balavishnuvj** [comment]
Hi @marcosvega91 , thank you reviewing so quick. Have addressed your concerns :)

**Review by marcosvega91 — DISMISSED** [review]
Lgtm

File: src/__tests__/screen.js [hunk 2]
```diff
@@ -21,6 +21,49 @@ test('exposes queries that are attached to document.body', async () => {
...
+  expect(console.log).toHaveBeenCalledTimes(1)
+  expect(console.log.mock.calls[0][0]).toMatchInlineSnapshot(`
+    "Open this URL in your browser
+
+    https://testing-playground.com/#markup=DwEwlgbgfAFgpgGwQewAQHdkCcEmAenGiA"
+  `)
+})
+
+test('logs messsage when element is empty', () => {
+  screen.logTestingPlaygroundURL(render('').container)
```
Comment (kentcdodds) [inline src/__tests__/screen.js:36]: No need to render here:

```suggestion
  screen.logTestingPlaygroundURL(document.createElement('div'))
```

**Review by kentcdodds — APPROVED** [review]
This is super! 👏 Just one little thing that I can merge myself. Thank you :)

**Review by kentcdodds — COMMENTED** [review]
@marcosvega91 reminded me that we should do TS updates on this PR. You up for that @balavishnuvj?

**Comment by balavishnuvj** [comment]
@kentcdodds I have tried to update the types. All the `t.ds` files were giving parsing error. So not sure if this defination works.

**Comment by balavishnuvj** [comment]
@all-contributors please add @balavishnuvj for code

**Pull request merged**